### ISSUE TITLE
[xas_viewer] Originally set fitting parameter cannot be valid

### ISSUE THREAD

**Issue by KanaTom** [xraypy/xraylarch#374]
Hello @newville ,

I always appreciate for your work on Larch.
In Feff Fitting tab, the parameters I originally defined is not used in fitting, even though vary=True in Fit script.
I'm trying to fit two path with different amplitude, so I change "s02" to "s02_1" and "s02_2" for Amplitude in respective path. After fitting, these parameters have never changed in recent try.

I checked if there is not too much parameters with artemis, and problem above never happened when I was using larch in beginning of 2022.
I hope my question is making sense for you.

![larch issue](https://user-images.githubusercontent.com/106894619/172031735-d7921081-e9a0-4f6c-b007-814be9601df1.PNG)

**Comment by newville** [comment]
@KanaTom Sorry for the trouble.  I think it is related to  https://millenia.cars.aps.anl.gov/pipermail/ifeffit/2022-May/010448.html and is related to a bug I introduced that basically meant the Path definitions were not being updated once they were initially created.

I believe this is fixed in the development version (that is, changing variables in Path Parameter definitions does work for me), which I think will be released (maybe with a couple of unrelated fixes) as 0.9.62 tomorrow or Tuesday).  If you are willing to try that, on windows, you should be able to open an "Anaconda Prompt (xraylarch)" terminal and run:

```
   python -m pip install https://millenia.cars.aps.anl.gov/xraylarch/downloads/xraylarch-latest-py3-none-any.whl
```

**Comment by KanaTom** [comment]
Thank you so much for your kind corresponding. Such fitting with too many parameters works very well with larch GUI.

I really appreciate for your work, and keep working with Larch with lab mates.
